packages/fuels-core/src/types/wrappers/coin.rs

```diff
@@ -26,27 +29,29 @@ pub struct Coin {
 
 impl From<ClientCoin> for Coin {
     fn from(coin: ClientCoin) -> Self {
+        let owner: Address = coin.owner;
         Self {
-            amount: coin.amount.0,
-            block_created: coin.block_created.0,
-            asset_id: coin.asset_id.0 .0,
-            utxo_id: coin.utxo_id.0 .0,
-            maturity: coin.maturity.0,
-            owner: coin.owner.0 .0.into(),
+            amount: coin.amount,
+            block_created: coin.block_created,
+            asset_id: coin.asset_id,
+            utxo_id: coin.utxo_id,
+            maturity: coin.maturity,
+            owner: Bech32Address::from(owner),
```

Comment:
forgot it before. sorry

```suggestion
            owner: Bech32Address::from(coin.owner),
```